app/assets/stylesheets/application.css.scss

```diff
@@ -26,7 +26,9 @@
   }
 
   li {
-    background-color: #f00;
+    background-color: #faebd7;
```

Comment:
Color literals like `#faebd7` should only be used in variable declarations; they should be referred to via variable everywhere else.